### DISCUSSION TITLE
Assigning multiple people to an issue

### DISCUSSION THREAD

**Discussion by Schafknuddler** [community/community#89798] [Codespaces] [answered]
### Select Topic Area

Question

### Body

Hey, when I try to assign multiple people to an issue, it shows them all for a second and then only shows one and removes the others. Dose that have something to do with using an organisation? Since before I moved the repositories to my organisation, it worked I think.

**Comment by mwenku** [comment]
The issue may be related to organization permissions, settings or user access controls, checking and adjusting these permissions in your organization.
could you clarify on your question ?

**Reply by Schafknuddler** [reply]
Where do I find such setting? Is it even there?

**Reply by mwenku** [reply]
one to specifically prevent multiple assignees doesn't seem to exist. https://stackoverflow.com/questions/76300555/how-to-restrict-github-issues-to-one-assignee mentions using an action to remove the additional users, you don't happen to have one do you ?

**Reply by Schafknuddler** [reply]
I didn't do anything.

**Reply by Schafknuddler** [reply, marked as answer]
I found the reason:
![image](https://github.com/community/community/assets/98185456/2767fdee-bcef-4e12-a755-4171bbc6cc0a)
It is only allowed for public repos until I upgrade my plan.
